### PR TITLE
APIGOV-17201 Update CLI Kubernetes install steps, and resources used by the Kubernetes discovery agent

### PR DESCRIPTION
**Pull request by tsjohns9**
Thank you for your contribution to the Axway-Open-Docs repo.

## Describe the changes

Updated CLI install steps for the Kubernetes agent to reflect the change from asking for a K8SCluster, to asking for an Environment.

Updated description of the resources used by the Kubernetes discovery agents to reflect the new resources added.

## Deploy preview link

Please add the deploy preview link to the **specific page** that you've changed.

_Although Netlify will add this link below, it will redirect to the main page of the documentation, not to an specific page._

## Checklist for contributors

Before submitting this PR, please make sure:

* [x] You have read the [contribution guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/)
* [x] You have signed the [Axway CLA](https://cla.axway.com/)
* [x] You have verified the technical accuracy of your change
* [x] You have verified that your change does not expose any sensitive information (passwords, keys, etc.)
* [ ] You have followed the [Markdown guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/writing_markdown/)  (unless this is is a Netlify CMS contribution)
* [ ] You have verified that all status checks have passed

_Put an x in the boxes that apply. This is simply a reminder of what we are going to look for before merging your change._

### CODE REVIEW

File: content/en/docs/central/mesh_management/deploy-your-agents-with-the-amplify-cli.md [hunk 1]
```diff
@@ -218,28 +218,28 @@ To use an existing DOSA account for the mesh agents, follow these steps:
     Created agent-secrets in the apic-control namespace.
     ```
 
-## Provide a K8S cluster resource
+## Provide an Environment resource
 
-After the details of the DOSA account have been provided you will be asked to either create a new K8S Cluster resource or provide the name of an existing K8S Cluster resource.
+After the details of the DOSA account have been provided you will be asked to either create an Environment resource or provide the name of an existing Environment resource.
```
Comment (markbice) [inline content/en/docs/central/mesh_management/deploy-your-agents-with-the-amplify-cli.md:223]: I think we can refer to Central a little more here... maybe "... asked to either create a new Environment resource in Amplify Central or provide the name of an existing Environment resource to hold your published Kubernetes cluster resources monitored and published by your Discovery agent.

File: content/en/docs/central/mesh_management/discover-apis-and-services.md [hunk 2]
```diff
@@ -65,13 +65,16 @@ When a pod or service is deleted in Kubernetes, the agents check if the deleted
...
-4. **APISpec** - A resource created by the ADA to represent unique API documentation found in a cluster. The same documentation may be found across multiple pods, especially if you are running multiple instances of one pod. One APISpec is created for one unique documentation found in a cluster. If multiple pods expose the same documentation, they will be grouped together in one APISpec. APISpecs are scoped to a K8SCluster and have a reference to a `SpecDiscovery`. APISpecs are created by the ADA as a response to finding a pod that exposes documentation as described by the `SpecDiscovery`.
-5. **K8SResource** - A resource created by the RDA that represents a pod or service discovered in Kubernetes. The K8SResource is scoped to a K8SCluster, and it has a reference to a ResourceDiscoverry. K8SResources are created by the RDA as a response to finding a pod or service that meets the match criteria described by the `ResourceDiscovery`.
+The discovery agents use eight resources:
+
+1. **Environment** - Resource representing an Amplify Central Environment
+2. **Mesh** - Represents a mesh, and has a reference to an Environment. Multiple K8SClusters can reference a Mesh.
```
Comment (markbice) [inline content/en/docs/central/mesh_management/discover-apis-and-services.md:71]: "Mesh" - "Represents a mesh collection of services and has a reference to an Environment. Multiple ..."

File: content/en/docs/central/mesh_management/discover-apis-and-services.md [hunk 3]
```diff
@@ -65,13 +65,16 @@ When a pod or service is deleted in Kubernetes, the agents check if the deleted
...
+3. **MeshDiscovery** - Used to promote services discovered in a mesh to an Environment.
```
Comment (markbice) [inline content/en/docs/central/mesh_management/discover-apis-and-services.md:71]: I think capitalize Mesh in this line

**Review by markbice — APPROVED** [review]
have reviewed and made suggestions Trevor has incorporated.

**Comment by andreamussap** [comment]
> have reviewed and made suggestions Trevor has incorporated.

@tsjohns9 @markbice If all comments were addressed can one of you please close the open comments/ suggestions by clicking 'Resolve comments'. Thank you

**Review by andreamussap — APPROVED** [review]

**Pull request merged**